Modules/ProfileManager/Profile.lua

```diff
@@ -48,6 +58,30 @@ function Profile:ClearMain()
     self.main = ""
 end
 
+function Profile:Alts()
+    return self.altList
+end
+
+function Profile:AltCount()
```

Comment:
The number of alts is not used anywhere; instead it is used as a boolean.
```suggestion
function Profile:HasAlts() 
    return not rawequal(next(self.alts), nil)
end
```